package.json

```diff
@@ -1,50 +1,57 @@
 {
   "name": "skynet-webportal",
+  "description": "Sia Skynet",
   "version": "1.0.0",
-  "homepage": "https://siasky.net",
+  "author": "Nebulous",
   "dependencies": {
-    "@testing-library/dom": "^6.12.2",
-    "@testing-library/jest-dom": "^5.1.1",
-    "@testing-library/react": "^9.3.2",
-    "@testing-library/user-event": "^10.0.0",
     "classnames": "^2.2.6",
+    "gatsby": "^2.19.7",
+    "gatsby-image": "^2.2.39",
+    "gatsby-plugin-manifest": "^2.2.39",
+    "gatsby-plugin-offline": "^3.0.32",
+    "gatsby-plugin-react-helmet": "^3.1.21",
+    "gatsby-plugin-sass": "^2.1.28",
+    "gatsby-plugin-sharp": "^2.4.3",
+    "gatsby-source-filesystem": "^2.1.46",
+    "gatsby-transformer-sharp": "^2.3.13",
     "jsonp": "^0.2.1",
     "node-sass": "^4.13.1",
     "prop-types": "^15.7.2",
     "react": "^16.12.0",
     "react-countup": "^4.3.3",
     "react-dom": "^16.12.0",
     "react-dropzone": "^10.2.1",
+    "react-helmet": "^5.2.1",
     "react-mailchimp-form": "^1.0.2",
     "react-mailchimp-subscribe": "^2.1.0",
     "react-reveal": "^1.2.2",
-    "react-scripts": "^3.4.0",
     "react-syntax-highlighter": "^12.2.1",
     "react-visibility-sensor": "^5.1.1",
     "shortid": "^2.2.15"
   },
   "devDependencies": {
     "prettier": "^1.19.1"
   },
+  "keywords": [
```

Comment:
perhaps add "decentralized" and "cloud storage"

Reply:
Good catch.